packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx

```diff
@@ -34,18 +33,19 @@ const WebsiteLinkCard = ({ link }) => {
       <Divider className="u-ml-0 u-maw-100" />
       <Media className="u-mt-1" align="top">
         <Img className="u-pr-1">
-          <Icon icon={GlobeIcon} color={palette['coolGrey']} />
+          <Typography color="textSecondary">
+            <Icon icon={GlobeIcon} />
+          </Typography>
         </Img>
         <Bd>
-          <ButtonLink
-            subtle
+          <Link
```

Comment:
Shouldn't the label be capitalized? This is the goal of ButtonLink, no?

Reply:
The label is not capitalized at the moment, see this screenshot.

<img width="169" alt="image" src="https://user-images.githubusercontent.com/465582/113993151-6ab73b00-9854-11eb-8735-ee1d950da0b4.png">

Reply:
Oh ok, it was right in front of my eyes https://github.com/cozy/cozy-libs/blob/ef670f968b0ef7d0550db3ff5eb8a7d4100bc66c/packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx#L14 🤦